pulsar-client-reactive-adapter/src/test/java/org/apache/pulsar/reactive/client/internal/adapter/AdaptedReactiveMessageConsumerTests.java

```diff
@@ -406,4 +407,26 @@ void consumePulsarException() throws Exception {
 			.verifyError(PulsarClientException.InvalidMessageException.class);
 	}
 
+	@Test
+	void closeConsumerExceptionIsIgnored() throws Exception {
```

Comment:
Excellent tests @lhotari - as usual 😸 . I commented out the fix and made sure they broke hehe